integration/Dockerfile

```diff
@@ -0,0 +1,25 @@
+FROM golang AS builder
```

Comment:
💡 is this file moved or completely new? What is it trying to achieve? (I think we have one Dockerfile already in place)

Reply:
I was adding some unit tests earlier and tried testing it using a different Dockerfile. This new Dockerfile should not be merged to master.